lib/build/recipe/emailverification/types.d.ts

```diff
@@ -90,16 +90,16 @@ export declare type RecipeInterface = {
               status: "EMAIL_VERIFICATION_INVALID_TOKEN_ERROR";
           }
     >;
-    isEmailVerified(input: { userId: string; email: string; userContext: any }): Promise<boolean>;
+    isEmailVerified(input: { recipeUserId: string; email: string; userContext: any }): Promise<boolean>;
     revokeEmailVerificationTokens(input: {
-        userId: string;
+        recipeUserId: string;
         email: string;
         userContext: any;
     }): Promise<{
         status: "OK";
     }>;
     unverifyEmail(input: {
-        userId: string;
+        recipeUserId: string;
         email: string;
         userContext: any;
```

Comment:
need to make sure that email verification has recipeUserId everywhere and not just userId